packages/blocks/src/page-block/edgeless/components/block-portal/frame/edgeless-frame.ts

```diff
@@ -54,6 +54,12 @@ export class EdgelessFramesContainer extends WithDisposable(LitElement) {
   @property({ attribute: false })
   surface!: SurfaceBlockComponent;
 
+  override connectedCallback(): void {
+    super.connectedCallback();
+    this.style.position = 'absolute';
```

Comment:
Can this be declared as static CSS?

Reply:
okay, let me try whether removing `createRenderRoot` will bring impact.
https://github.com/toeverything/blocksuite/blob/6ba0419a97e5977093cf22e985aa47a332de6047/packages/blocks/src/page-block/edgeless/components/block-portal/frame/edgeless-frame.ts#L72C1-L74C4

Reply:
because this
https://github.com/toeverything/blocksuite/blob/6ba0419a97e5977093cf22e985aa47a332de6047/packages/blocks/src/frame-block/frame-block.ts#L39-L41
in order to access parentBlockElement, we can't put blockelement in shadow-root. so static styles will not work.

Reply:
In this case, why not use `ShadowlessElement`?